backend/src/main/java/dccs/academy/services/SupplierService.java

```diff
@@ -0,0 +1,23 @@
+package dccs.academy.services;
+
+import dccs.academy.dtos.SupplierDto;
+import dccs.academy.entities.SupplierEntity;
+import dccs.academy.repositories.SupplierRepository;
+import jakarta.enterprise.context.ApplicationScoped;
+import jakarta.inject.Inject;
+import jakarta.transaction.Transactional;
+
+@ApplicationScoped
+@Transactional
+public class SupplierService {
+    @Inject
+    SupplierRepository supplierRepository;
+    public SupplierDto createSupplier(SupplierDto supplierDto){
+        var supplierEntity= new SupplierEntity();
+        supplierEntity.setNames(supplierDto.getNames());
```

Comment:
In general, you would want to have a mapper class that does mapping between dto and entity. Reason is that way you extract it in function and you can call it whenever and also it does not make your service code unnecessary complex.